Python/Product/Analysis/LanguageServer/Server.WorkspaceSymbols.cs

```diff
@@ -50,6 +51,14 @@ await GetModuleVariablesAsync(entry as ProjectEntry, opts, @params.query, 50)
                 .ToArray();
         }
 
+        public async Task<DocumentSymbol[]> NewDocumentSymbol(DocumentSymbolParams @params, CancellationToken cancellationToken) {
```

Comment:
Public?

Reply:
Ah, I see, this is the newer one. Personally I'd prefer a suffix `2` or `3_10` (for the LSP version), or a descriptive prefix (`HierarchyDocumentSymbol`?)

Reply:
Sure